src/vs/workbench/contrib/mergeEditor/browser/utils.ts

```diff
@@ -53,21 +53,21 @@ export class ReentrancyBarrier {
 	}
 }
 
-export function n<TTag extends string>(tag: TTag): never;
-export function n<TTag extends string, TId extends string>(
+export function h<TTag extends string>(tag: TTag): never;
```

Comment:
Cryptic

Reply:
Joh suggested to rename this function to `h` (like `html`). `n` was referring to `node`.

I will move it to some global utils as next step and encourage its usage.

Reply:
Yeah 👍